packages/css-library/tokens/fonts.json

```diff
@@ -1,12 +1,12 @@
 {
   "font": {
     "family": {
-      "sans": { "value": "'Source Sans Pro', 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif" },
+      "sans": { "value": "'Source Sans Pro Web', 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif" },
       "serif": { "value": "Bitter, Georgia, Cambria, 'Times New Roman', Times, serif" }
     },
     "serif": { "value": "Bitter, Georgia, Cambria, 'Times New Roman', Times, serif" },
     "source": {
-      "sans": { "value": "'Source Sans Pro', 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif"}
+      "sans": { "value": "'Source Sans Pro Web', 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif"}
```

Comment:
our utility classes needed to be updated with the newly imported `base/fonts` because of the uswds rename to `Source Sans Pro Web`. Without this addition the utility class `vads-u-font-family--sans` will render a serif font due to not finding the right font-face.